src/renderer/constants/settings.js

```diff
@@ -11,4 +11,6 @@ export const INSTANT_PURCHASE_ENABLED = 'instantPurchaseEnabled';
 export const INSTANT_PURCHASE_MAX = 'instantPurchaseMax';
 export const THEME = 'theme';
 export const THEMES = 'themes';
+export const DARK_THEME = 'dark';
+export const LIGHT_THEME = 'light';
```

Comment:
Currently these values are setting _keys_ not setting _values_. It's a good idea to const these theme values, but we ought to do this in a way that retains that distinction.